core/templates/dev/head/profile/profile.html

```diff
@@ -49,13 +49,224 @@ <h1 class="oppia-profile-username"><strong>{{PROFILE_USERNAME}}</strong></h1>
+                                          </span>
+                                          <span ng-if="exploration.objective" class="hidden-xs">
+                                            <span style="font-weight: bold;">&middot;</span>
+                                            <span style="color: #888; font-size: 0.9em;" class="protractor-test-exploration-objective">
+                                              <[exploration.objective | truncateAndCapitalize: 120]>
+                                            </span>
+                                          </span>
+                                        </div>
+
+                                        <div class="oppia-gallery-tile-second-row">
+                                          <span style="color: #888">
+                                            <strong><[exploration.category]></strong>
+                                            <span style="font-weight: bold;">&middot;</span>
+                                            Last updated <[getLocaleAbbreviatedDatetimeString(exploration.last_updated)]>
+                                          </span>
+                                          <span ng-if="exploration.status === 'publicized'">
+                                            <span style="font-weight: bold;">&middot;</span>
+                                            <span style="color: #888">Featured</span>
+                                          </span>
+                                          <span ng-if="areRatingsShown(exploration.ratings)">
+                                            <span style="font-weight: bold;">&middot;</span>
+                                            <span class="oppia-gallery-tile-ratings" style="color: black;">
+                                              <rating-from-frequencies rating-frequencies="exploration.ratings"></rating-from-frequencies>
+                                            </span>
+                                          </span>
+                                        </div>
+                                      </div>
+                                    </div>
+                                  </a>
+                                </md-card>
+                              </div>
+                                     
+                                   </span>
+                                   <span ng-if="!userPlaysOnCreatedExploration">
+                                        <em>This user hasn't created any explorations yet.</em>
+                                    </span>
+                                  </p>
+                        </md-content>
+                    </md-tab>
+                </md-tabs>
+            </md-content>
+        </div>
     </div>
   </div>
+                              
+{% include 'components/ratings.html' %}
 {% endblock %}
 
+
+
 {% block footer_js %}
   {{ super() }}
   <script>
     {{ include_js_file('/profile/Profile.js') }}
```

Comment:
can you remove the leading slash here? (I know this was in the code before; I figured we might as well be consistent.)